source/common/config/metadata.h

```diff
@@ -18,8 +18,9 @@
 namespace Envoy {
 namespace Config {
 
-using ConstMetadataSharedPoolSharedPtr = std::shared_ptr<
-    SharedPool::ObjectSharedPool<const envoy::config::core::v3::Metadata, MessageUtil>>;
+using ConstMetadataSharedPoolSharedPtr =
+    std::shared_ptr<SharedPool::ObjectSharedPool<const envoy::config::core::v3::Metadata,
+                                                 MessageUtil, MessageUtil>>;
```

Comment:
formatting change only?

Reply:
It's hard to read, but there's an extra template argument (see the repeated `MessageUtil`) for `EqualFunc`

Reply:
got it